README

```diff
@@ -926,6 +931,11 @@ Prerequisites
      `varnish' plugin.
      <http://varnish-cache.org>
 
+  * mcelog
```

Comment:
I don't think you need to add this block in this section, since it is mcelog is not a library and needed for building collectd.